### DISCUSSION TITLE
Number of Edge Functions calls

### DISCUSSION THREAD

**Discussion by eposha** [vercel/community#752] [Help] [answered]
Where can I see the number of Edge Functions I've called to see if I'm under the monthly limit?

**Comment by amyegan** [comment, marked as answer]
You can see [usage](https://vercel.com/docs/concepts/limits/usage) on your account at https://vercel.com/dashboard/usage

More info about Serverless and Edge Functions here https://vercel.com/docs/concepts/functions/edge-functions#key-takeaways
